internal/token_test.go

```diff
@@ -44,6 +44,11 @@ func TestBasic(t *testing.T) {
 			`<pkg.Item>`,
 			[]TokenType{StartTagToken},
 		},
+		{
+			"noscript component",
+			`<noscript><Component /></noscript>`,
+			[]TokenType{StartTagToken, SelfClosingTagToken, EndTagToken},
```

Comment:
Previously all contents inside of `noscript` were tokenized as `TextToken` instead of being tokenized as regular content.